api-gateway/security/README.md

```diff
@@ -0,0 +1,48 @@
+
+# API Gateway security example
+This project is a result of completing either [Getting Started with Knot.x Stack](http://knotx.io/tutorials/getting-started-with-knotx-stack/2_0/) or [Getting started with Docker](http://knotx.io/tutorials/getting-started-with-docker/2_0/) tutorials.
+
+This example is built using the [Starter Kit](https://github.com/Knotx/knotx-starter-kit) template project.
+
+## How to build & run
+
+#### Prerequisites
+- JDK 8
+- Docker
+
+#### Run
+```
+$ ./gradlew build-docker
+```
+
+The [build-docker](https://github.com/Knotx/knotx-starter-kit#build--validate-docker-image) builds and validates the custom project Docker image.
+
+Then, start Docker container:
+```
+$ docker run -p8092:8092 knotx/secure-api-gateway
+```
+
+and validate url:
+- [`localhost:8092/api/public`](http://localhost:8092/api/public)
+
+## Endpoints:
+
+- [`localhost:8092/api/public`](http://localhost:8092/api/public) public endpoint without security
+- [`localhost:8092/api/secure/basic`](http://localhost:8092/api/security/basic) endpoint protected with [Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication)
```

Comment:
I am wondering if this tutorial should not focus on the OAuth2 scenario. It can be misleading when we mix a basic auth, an API token and OAuth2.

Reply:
I've removed the public to simplify things, is that ok?